pkg/internal/checkup/executor/testpmd/console_test.go

```diff
@@ -0,0 +1,134 @@
+package testpmd_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	expect "github.com/google/goexpect"
+	assert "github.com/stretchr/testify/require"
+
+	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/executor/testpmd"
+)
+
+const (
+	vmiUnderTestEastNICPCIAddress = "0000:06:00.0"
+	trafficGenEastMACAddress      = "60:94:19:c9:ac:01"
+	vmiUnderTestWestNICPCIAddress = "0000:07:00.0"
+	trafficGenWestMACAddress      = "60:94:19:c9:ac:02"
+	verbosePrintsEnabled          = false
+)
+
+func TestGetPortStatsSuccess(t *testing.T) {
+	t.Run("when GetStats succeed", func(t *testing.T) {
```

Comment:
Please consider removing the sub-test if there is only one.

Reply:
DONE